### ISSUE TITLE
No emblems are shown on files or folders in Column View at the smallest zoom level

### ISSUE THREAD

**Issue by ghost** [elementary/files#1211]
# Prerequisites
- [x] I have searched open and closed issues for duplicates.

# Describe the error

No emblems are shown on files or folders in Column View at the smallest zoom level. By emblems I mean the small icons which indicate that a file has read only permissions, is inaccessible, is a link, etc.

I presume that this may also apply to [the recent addition of Git-related emblems](https://twitter.com/elementary/status/1216770409410039808), and other emblems which this test does not cover.

## To reproduce

1. Open Files.
2. Create the empty files “Essai1”, “Essai2”, and “Essai3”.
3. Open Terminal.
4. Run these commands: `sudo chgrp root Essai2; sudo chmod a=r Essai2; sudo chgrp root Essai3; sudo chmod a=x Essai3`
5. In the Home folder, copy Essai as a link and paste it as a link in the folder.
6. Essai2 has a lock, Essai3 has a red denial sign, and the link to Essai has a small arrow.
7. Display the files in Column View.
8. Zoom out to the smallest zoom level.
9. The emblems are not visible.

## Expected behaviour

Emblems are always shown on files and folders no matter the zoom level. In the case of the smallest zoom level, the emblems would simply be smaller.

If the lack of emblems is intentional, then allow me to present an argument for why this should be changed. It is currently difficult to view many files and folders at once in Files. The smallest zoom level in Column View is what grants one the greatest possible overview. However, to have this overview comes with the cost of not knowing whether a given file or folder is locked, one lacks sufficient permissions to access it, it is a link to another file or folder, or it carries some other status which an emblem would indicate. To me, this is an arbitrary and unnecessary limitation. A smaller icon, even if it would appear slightly blurry, would be much better than no icon at all. As it stands, the smallest zoom level in Column View is inadequate for regular use because one must switch to another view or zoom level in order to at a glance know whether a file or a folder has a special status attached to it.

## Screenshots or screen recordings

[Screen recording](https://github.com/elementary/files/files/4144311/No.emblems.are.shown.on.files.or.folders.in.Column.View.zip)

# Platform information

![System information](https://user-images.githubusercontent.com/51057001/73609855-d4ddab80-45d1-11ea-8202-a8b1432f6811.png)

- [ ] I'm using the latest version from git that I've manually compiled
- [x] I'm using the latest released stable version

Language: French
Keyboard: English (US) - English (Dvorak intl., with dead keys)
Location: Paris

The testing system is installed on a small partition on a ThinkPad T470 laptop on which UEFI Secure Boot is disabled. Aside from specifying the language, keyboard, location, initial user, and the specific partition in which to install elementary OS, all of the default settings were used during the installation. No Internet connection was used during the installation. The only system-wide changes which have been made after the installation are that the system connects to the Internet by wi-fi via a Linksys WRT54GL Wireless Router and that [Screen Recorder](https://appcenter.elementary.io/com.github.mohelm97.screenrecorder/) has been installed via AppCentre.

# Additional context

Before this test was performed, the following steps were followed:

1. All system-wide changes from previous tests were reverted.
2. All system components were updated via the blue “Update All” button in AppCentre.
3. A new administrator user named “Essai” or “Essai 2” was created with the password “komalleana”.
4. The current user was logged out and the new user was logged in.
5. The old user was deleted, the screen resolution was set to 800x600 at 60 Hz, tap clicking was disabled, and the touchpad pointer speed was set to the maximal value.
6. The computer was rebooted.
6. The test user was logged in.

**Comment by jeremypw** [comment]
This is intended behaviour to avoid the emblems obscuring the icon too much.

**Comment by jeremypw** [comment]
At the moment very small emblems are not available in the theme and the smallest available were considered too large for the smallest icons available.

**Comment by KomalleanaByzantis** [comment]
@jeremypw I suspected that this behaviour was intentional, hence why I present this argument in my report:

> It is currently difficult to view many files and folders at once in Files. The smallest zoom level in Column View is what grants one the greatest possible overview. However, to have this overview comes with the cost of not knowing whether a given file or folder is locked, one lacks sufficient permissions to access it, it is a link to another file or folder, or it carries some other status which an emblem would indicate. To me, this is an arbitrary and unnecessary limitation. A smaller icon, even if it would appear slightly blurry, would be much better than no icon at all. As it stands, the smallest zoom level in Column View is inadequate for regular use because one must switch to another view or zoom level in order to at a glance know whether a file or a folder has a special status attached to it.

Hiding the emblems does not solve the problem, but merely creates a new one in its place. Would it not be possible to create a set of smaller emblems? The smallest emblems are 16x16 pixels in size. Why not create a set of emblems which are 12x12 or 8x8 pixels in size? Then you could also use those emblems for the second lowest zoom level in Column View, where the file icons are already close to being overshadowed by the emblems. You would also be able to reuse those emblems in a hypothetical smaller zoom level for List and Grid View.

**Comment by jeremypw** [comment]
Yes, that would be a solution but you would need to raise the issue against https://github.com/elementary/icons I am afraid.

**Comment by jeremypw** [comment]
Closing as the emblems are now shown in-line in List and Column views, rather than as icon overlays and do not disappear at the smallest size.
